### PR TITLE
N40: Add bevy_atmosphere

### PR DESCRIPTION
**Pull request by JonahPlusPlus**
_Part of #1203_

This covers bevy_atmosphere's 0.5 update.

### CODE REVIEW

File: content/news/040/index.md [hunk 1]
```diff
@@ -124,6 +125,41 @@ You can check this [Reddit post][boytacean-red-ann] for more information.
...
+is the same model that was used in previous version of bevy_atmosphere.
+`Gradient` is a new model that provides a simple gradient of three colors,
+making it ideal for stylized games.
+
+There is also the `Atmosphere<T>` and `AtmosphereMut<T>` system params,
+which can be used to work with a particular model
+without having to cast it from `AtmosphereModel`.
+
+If you want to read more about the technical changes, check out my
+[blog post](https://jonahplusplus.dev/2022/12/01/bevy_atmosphere_0.5.html)!
```
Comment (17cupsofcoffee) [inline content/news/040/index.md:150]:
```suggestion
If you want to read more about the technical changes, check out the developer's
[blog post](https://jonahplusplus.dev/2022/12/01/bevy_atmosphere_0.5.html)!
```

File: content/news/040/index.md [hunk 2]
```diff
@@ -124,6 +125,41 @@ You can check this [Reddit post][boytacean-red-ann] for more information.
 
 ## Library Updates
 
+### [bevy_atmosphere 0.5](https://github.com/JonahPlusPlus/bevy_atmosphere)
+
+![bevy_atmosphere collage](bevy_atmosphere.png)
+
+bevy_atmosphere is now compatible with bevy 0.9.
```
Comment (17cupsofcoffee) [inline content/news/040/index.md:132]:
```suggestion
bevy_atmosphere ([crates.io](https://crates.io/crates/bevy_atmosphere), [docs.rs](https://docs.rs/bevy_atmosphere/latest/bevy_atmosphere/), [GitHub](https://github.com/JonahPlusPlus/bevy_atmosphere))
is now compatible with Bevy 0.9.
```

File: content/news/040/index.md [hunk 3]
```diff
@@ -124,6 +125,41 @@ You can check this [Reddit post][boytacean-red-ann] for more information.
...
+[blog post](https://jonahplusplus.dev/2022/12/01/bevy_atmosphere_0.5.html)!
+
+_External Links:
+[Blog Post](https://jonahplusplus.dev/2022/12/01/bevy_atmosphere_0.5.html),
+[crates.io](https://crates.io/crates/bevy_atmosphere),
+[docs.rs](https://docs.rs/bevy_atmosphere/latest/bevy_atmosphere/),
+[GitHub](https://github.com/JonahPlusPlus/bevy_atmosphere)_
+
+_Discussions:
+[/r/rust_gamedev](https://www.reddit.com/r/rust_gamedev/comments/za948o/bevy_atmosphere_05_is_now_released/),
```
Comment (17cupsofcoffee) [inline content/news/040/index.md:159]:
```suggestion

_Discussions:
[/r/rust_gamedev](https://www.reddit.com/r/rust_gamedev/comments/za948o/bevy_atmosphere_05_is_now_released/),
```

**Review by 17cupsofcoffee — APPROVED** [review]
:+1: LGTM, just some tweaks to style.

**Review by ozkriff — APPROVED** [review]
:+1: (optimized the image)

**Comment by ozkriff** [comment]
Merged, thanks!

**Pull request merged**